### ISSUE TITLE
Typescript / Eslint issue (no-undefined)

### ISSUE THREAD

**Issue by rafaelruda** [single-spa/create-single-spa#273]
This is just a issue report, as i already solved it locally.

Whenever i create a new root-config module-type project, with Typescript, the `.eslintrc` file starts with this configuration:
```
{
  "extends": ["important-stuff", "plugin:prettier/recommended"],
  "parser": "babel-eslint"
}
```

The problem is, `important-stuff` does not work for typescript, and create inumerous conflicts with eslint, like no-undefined references errors for type declarations.

I solved it changing `important-stuff` to `ts-important-stuff`. At least, it is what worked for me.

Hope this helps someone.

**Comment by joeldenning** [comment]
Thanks for the bug report. I have fixed this in #274

**Comment by joeldenning** [comment]
Fix released in https://github.com/single-spa/create-single-spa/releases/tag/v2.2.0
